### DISCUSSION TITLE
章鱼网络的原理是啥啊

### DISCUSSION THREAD

**Discussion by Enigma-Soul** [Meloong-Git/PCL#2642] [提问] [answered]
闲的...

**Comment by Pigeon0v0** [comment]
![image](https://github.com/Hex-Dragon/PCL2/assets/60414767/6f5e4ebc-39f7-4a5c-b715-80de2bf9082d)

**Reply by Pigeon0v0** [reply, marked as answer]
补充：P2P+Relay，等等看八月过去应该会做的

**Reply by Enigma-Soul** [reply]
打洞是吧...

**Reply by Pigeon0v0** [reply]
> 打洞是吧...
> 
> 

你知道就好.jpg

**Reply by Enigma-Soul** [reply]
我还以为是什么不需要公网服务器的东西呢...
